### PR TITLE
Feature postpone expiration #87

### PR DESCRIPTION
**Pull request by verheyenkoen**
Postpone folder expiration (issue #87)

### CODE REVIEW

File: handlers/folders.go [hunk 1]
```diff
@@ -99,6 +99,29 @@ func UpdateFolder(w http.ResponseWriter, r *http.Request) {
...
+	c := ctx.Get(r)
+	folder := ctx.GetFolder(r)
+
+	folder.PostponeExpiration()
+
+	if err := c.Repo.Folders.Update(r.Context(), folder); err != nil {
+		c.PersistFlash(w, ctx.Flash{
+			Type:         "error",
+			Body:         fmt.Sprintf("Unexpected error: %s", err.Error()),
+			DismissAfter: 0,
```
Comment (nics) [inline handlers/folders.go:112]: 0 is the default value, so this line is not needed

File: models/folder.go [hunk 2]
```diff
@@ -13,6 +13,8 @@ import (
 
 var reSlug = regexp.MustCompile("[^a-zA-Z0-9-]+")
 
+const PostponeByDays = 31
```
Comment (nics) [inline models/folder.go:16]: It's better to use a `time.Duration`: `const PostPonePeriod = 31*24*time.Hour`

And then further down: `ExpiresAt: created.Add(PostponePeriod)`

**Review by nics — CHANGES_REQUESTED** [review]
looking good, just a few niggles

File: assets/js/app.js [hunk 3]
```diff
@@ -10,47 +9,63 @@ import selectValue from './select_value.js'
 window.htmx = htmx
 
 // load htmx extensions
-require('htmx.org/dist/ext/ws.js');
+require('htmx.org/dist/ext/ws.js')
```
Comment (nicolasfranck) [inline assets/js/app.js:12]: try not to reset all padding. That makes a PR harder to read ;-)

Comment (verheyenkoen) [reply assets/js/app.js:12]: Using Prettier with default style rules. I'll keep it in mind.

Comment (nics) [reply assets/js/app.js:12]: No it's good that you introduced this, we had formatting for Go code, but not yet for js

Comment (nics) [reply assets/js/app.js:12]: I'm going to use the same rules from now on

Comment (verheyenkoen) [reply assets/js/app.js:12]: @netsensei Probably best to add Prettier as a devcontainer extension then

Comment (verheyenkoen) [reply assets/js/app.js:12]: I found I had user preferences that overruled certain defaults. Will recommit this file with default formatting rules applied.

**Review by nics — APPROVED** [review]

**Pull request merged**